_locales/en/messages.json

```diff
@@ -158,10 +158,13 @@
       "description": "Label for a button to accept a new safety number"
     },
     "verify": {
-      "message": "Verify"
+      "message": "Mark as verified"
+    },
+    "unverify": {
+      "message": "Clear verification"
```

Comment:
That's fair. But I will submit that 'Clear verification' is what iOS uses, and it's the only other app with a button for this action. Android has that switch with a simple label 'Verified.'